app/src/main/res/values/style.xml

```diff
@@ -6,6 +6,12 @@
         <item name="color">@color/theme_bright</item>
     </style>
 
+    <style name="profileListCirclePic" parent="">
+        <item name="cornerFamily">rounded</item>
+        <item name="cornerSize">30dp</item>
+        <item name="color">@color/theme_bright</item>
```

Comment:
Did you look if you could use an ?attr ?

Reply:
No but it is not too important here